assets/css/main.scss

```diff
@@ -4,5 +4,15 @@
 
 @charset "utf-8";
 
+
+// define variable with front matter
+{% if site.data.data.sidebar.position == 'left' %}
```

Comment:
@sharu725 I think in the case sidebar.position is not defined, this would be false, and the sidebar is going to be on the right side :thinking:

Reply:
I agree with you, I will test ASAP to be sure ;)